src/main/resources/db/migration/V112__add_new_table_for_find_out_more.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE public.search_additionallink (
+    id integer PRIMARY KEY NOT NULL,
+    url character varying(255),
```

Comment:
Should these be not null? Is it weird to have (potentially) a url/description that are both null coming back? Maybe best to protect against it here. Is there a case where this may be on a tab on the admin portal in the future? If so, then it increases the risk 🦜

Reply:
Yeah it is better to make them not null. I will add them